baldrick/plugins/milestone_checker.py

```diff
@@ -0,0 +1,33 @@
+from flask import current_app
+
+from .pull_request_checker import pull_request_check
+
+
+MISSING_MESSAGE = 'This pull request has no milestone set.'
+PRESENT_MESSAGE = 'This pull request has a milestone set.'
+
+
+@pull_request_check
+def process_milestone(pr_handler, repo_handler):
+    """
+    A very simple set a failing status if the milestone is not set.
+    """
+    if not repo_handler.get_config_value("check_milestone", False):
+        return [], None
+
+    fail_message = repo_handler.get_config_value("milestone_checker", {}).get("missing_message", MISSING_MESSAGE)
```

Comment:
General comment on readability, maybe we should change ``get_config_value`` to be just ``config`` and make it a cached property. Then this would read ``repo_handler.config.get(...)`` - just reduces the number of times we say 'get'